data-notes.md

```diff
@@ -1,3 +1,4 @@
+```
```

Comment:
This was a manual edit... should this file be deleted?

Reply:
yeah, it's really old file with some notes I made early in v1.